README.md

```diff
@@ -57,3 +57,5 @@ Other ASS/SSA implementations:
 - [Perian](https://github.com/MaddTheSane/perian)
 - [asa](http://git.spaceboyz.net/asa.git) (defunct)
 - [libjass](https://github.com/Arnavion/libjass)
+- [SubtitlesOctopus](https://github.com/Dador/JavascriptSubtitlesOctopus) (builds on wasm libass)
+- [Javascript ASS](https://github.com/weizhenye/ASS)
```

Comment:
It seems the name is ASS.js, and it seems it doesn’t actually use libass.

Reply:
SubtitlesOctopus does pull in libass as a dependency.

Reply:
I can also move it up the list, which is technically more correct. But fuctionally SubtitlesOctopus makes libass available for user on browsers, which makes it use-case more similar to ASS.js and libjass.

Reply:
I’m _slightly_ inclined to think it should go in the first list because it’s more “software that uses libass” than “another ASS implementation”, but I do appreciate that it’s kinda middle ground. Would be good to hear other opinions, but ultimately this isn’t important.

Reply:
The above was only my reasoning for putting it to the ASS implementations, but I don't really have any strong opinion on this. I now moved SubtitleOctopus up to "software that uses libass".